### PR TITLE
Clarify FAQ permissions tip

### PR DESCRIPTION
**Pull request by twerq**
Clarify FAQ permissions tip for those who not on Debian

### CODE REVIEW

**Comment by twerq** [comment]
I've spent about 3 hours in investigation of this. Maybe it'd save at least one man-month for us :-)

File: docs/en/faq.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ For other OS's, follow the instructions in the "How to make ZeroNet work with To
  - `sudo nano /etc/tor/torrc`
  - Remove the `#` character from lines `ControlPort 9051` and `CookieAuthentication 1` (line ~57)
  - Restart tor
- - Add permission for yourself to read the auth cookie. With Debian Linux, the command is `sudo usermod -a -G debian-tor [yourlinuxuser]`<br>(if you are not on Debian check the file's user group by `ls -al /var/run/tor/control.authcookie`)
+ - Add permission for yourself to read the auth cookie. With Debian Linux, the command is `sudo usermod -a -G debian-tor [yourlinuxuser]`<br>(if you are not on Debian check the file's user group by `ls -al /var/run/tor/control.authcookie` and check directory permissions by `ls -ld /var/run/tor`)
```
Comment (Mikaela) [inline docs/en/faq.md]:
```suggestion
 - Add permission for yourself to read the auth cookie. With Debian Linux, the command is `sudo usermod -a -G debian-tor [yourlinuxuser]`<br>(if you are not on Debian check the file's user group by `ls -al /var/run/tor/`)
```

if you specify only the directory, you will also see the permissions of the files in it.

Comment (twerq) [reply docs/en/faq.md]: Yes, I see. And I believe we'd to draw some attention to dir permissions specifically, not only to it ownership. In my case after running tor service with described settings `/var/run/tor` perm were `drwx--S---` and should be `drwx--s---`.

**Comment by HelloZeroNet** [comment]
I think it would be better to put it on separate line, eg.:
```markdown
> __Tip:__ Use the `ls -ld /var/run/tor` command to make sure it has the correct `drwxr-sr-x` permission bits. (fix it with `chmod g+sx /var/run/tor/` if necessary)
```

**Pull request merged**